scripts/scr_ui_manage/scr_ui_manage.gml

```diff
@@ -1,46 +1,69 @@
 function scr_ui_manage() {
+	
+	var romanNumerals=[
```

Comment:
I'd extract this definition to a separate script, so we won't have to repeat it in multiple scripts